src/components/screens/multiSignature/editor/editor.js

```diff
@@ -7,16 +7,36 @@ import TransactionPriority from '../../../shared/transactionPriority';
 import { tokenMap } from '../../../../constants/tokens';
 import useTransactionFeeCalculation from '../../send/form/useTransactionFeeCalculation';
 import useTransactionPriority from '../../send/form/useTransactionPriority';
+import { PrimaryButton, TertiaryButton } from '../../../toolbox/buttons';
+import { Input } from '../../../toolbox/inputs';
 
+import MemberField from './memberField';
 import styles from './styles.css';
-import { PrimaryButton } from '../../../toolbox/buttons';
+import ProgressBar from '../progressBar';
 
 const token = tokenMap.LSK.key;
-const txType = 'multiSignature';
+const txType = 'createMultiSig';
+const MAX_MULTI_SIG_MEMBERS = 64;
 
+const placeholderMember = {
+  identifier: undefined, isMandatory: false,
+};
+
+// eslint-disable-next-line max-statements
```

Comment:
What do you think about breaking down this component like `/screens/lockedBalance/lockedBalance` in 2, `editor` `form`? It is already exceeding max-statements, that would help it to make it more readable.

Reply:
It would not make much of a difference because all of the state will still be in the parent and we will only be able to more some jsx to a new component.